src/Config.hpp

```diff
@@ -398,12 +398,18 @@ inline void
 Config::set_base_dir(const std::string& value)
 {
   m_base_dir = value;
+#ifdef _WIN32
+  std::replace(m_base_dir.begin(), m_base_dir.end(), '\\', '/');
+#endif
```

Comment:
`Config::set_base_dir` is only called by test code, so this change won't have any effect on the base dir value set via configuration file or environment variable. Is that what you intended?

I guess the idea is to normalize `base_dir` to only use forward slashes? But if we do that then it won't be possible to match paths (in `Util::make_relative_path`) that use backslashes. That is, if we normalize `base_dir`, shouldn't we also normalize the paths that are compared with `base_dir`?